ement-room.el

```diff
@@ -2415,17 +2412,21 @@ To be called from a minibuffer opened from
                        ;; TODO: Use letrec with Emacs 27.
                        (remove-hook 'minibuffer-exit-hook compose-fn-symbol)
                        (ement-room-compose-message ement-room ement-session :body body)
-                       (set-input-method input-method) ; Set in compose buffer.
                        (setf ement-room-send-message-filter send-message-filter)
                        (let* ((compose-buffer (current-buffer))
                               (show-buffer-fn-symbol (gensym "ement-show-compose-buffer"))
                               (show-buffer-fn (lambda ()
                                                 (remove-hook 'window-configuration-change-hook show-buffer-fn-symbol)
-                                                (pop-to-buffer compose-buffer))))
+                                                (pop-to-buffer compose-buffer)
+                                                (set-input-method input-method))))
                          (fset show-buffer-fn-symbol show-buffer-fn)
                          (add-hook 'window-configuration-change-hook show-buffer-fn-symbol)))))
     (fset compose-fn-symbol compose-fn)
     (add-hook 'minibuffer-exit-hook compose-fn-symbol)
+    ;; If minibuffer's input method is not deactivated, then all
+    ;; minibuffer commands inherit the room's input method.  This is
+    ;; not the correct behaviour.
+    (set-input-method nil)
```

Comment:
Good catch, thanks.  We should probably use `deactivate-input-method` though; according to Emacs docs, that's the right way.

Reply:
I will do that.